Earthfile

```diff
@@ -35,10 +35,30 @@ minikube:
     RUN chmod +x minikube
     SAVE ARTIFACT minikube /binary
 
+python-requirements:
```

Comment:
I think I missed where this gets used

Reply:
That was before I realized we had an `ansible` target...